src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/SingleColumnTableCoercion.txt

```diff
@@ -0,0 +1,486 @@
+// text coercions
+
+>> Cos(["3.2","2.5","1.9"])
+Table({Value:-0.9982947757947531},{Value:-0.8011436155469337},{Value:-0.32328956686350335})
```

Comment:
we should add a comment that calculations aren't done with decimal but double type
otherwise it should be `Table({Value:-0.9982947757947530846616607223},{Value:-0.8011436155469337148335027905},{Value:-0.3232895668635034222788336951})`
(+1 on Carlos' comment. Keep all digits except last 3 ones.)